src/document/url.mli

```diff
@@ -27,6 +27,9 @@ module Path : sig
 
   val string_of_kind : kind -> string
 
+  val pp_kind_prefix_for_output : Format.formatter -> kind -> unit
+  (** Print the ["kind-"] prefix used in output files. *)
```

Comment:
This seems to be for output file in html flat mode, but if I understand correctly used as labels in latex mode (and not as filenames).

Maybe:

```suggestion
  val pp_disambiguating_prefix : Format.formatter -> kind -> unit
  (** Print the ["kind-"] prefix used to disambiguate urls in "flat modes": e.g. latex labels and output files in [--flat] HTML output *)
```

Reply:
And the man output too. Thanks!